src/log/entry.cr

```diff
@@ -8,6 +8,20 @@ enum Log::Severity
   Fatal
   # Used only for severity level filter.
   None
+
+  def label
+    case self
+    when Debug  ; "DEBUG"
+    when Verbose; "VERBOSE"
+    when Info   ; "INFO"
+    when Warning; "WARNING"
+    when Error  ; "ERROR"
+    when Fatal  ; "FATAL"
+    when None   ; "NONE"
+    else
+      raise NotImplementedError.new("Log::Severity#label")
```

Comment:
It's not really relevant, but I don't think should use `NotImplementedError`. This should just be `raise "unreachable"`.

Reply:
Better yet: this shouldn't be needed at all, cough, cough.